src/index.tsx

```diff
@@ -1,3 +1,5 @@
+import 'react-app-polyfill/ie9';
```

Comment:
No, don't support Internet Explorer!  Isn't it end of life now anyways?

Reply:
Yeah it seems unnecessary to support IE. Retracted this.